curriculum/js1/4.js

```diff
@@ -6,11 +6,11 @@
  * @returns {string}
  */
 
-const solution = (n, b) => {
-  if (n===1) {
-    return b;
+const solution = (n, b, c = b) => {
+  if (n === 1) {
```

Comment:
Recommendation: use `n >= 1` to handle cases where `solution` is called with an `n ` value of `0`